src/index.ts

```diff
@@ -76,18 +77,26 @@ export default function loader(
     rootContext,
     resourcePath,
     resourceQuery: _resourceQuery = '',
+    _compiler,
   } = loaderContext
 
+  const webpackVersion = _compiler?.webpack?.version
+  const isWebpack5 = webpackVersion && webpackVersion[0] > '4'
```

Comment:
webpackVersion[0] > '4' is dangerous when webpackVersion > 10.x.y, I thinks direct equal test should be better

Reply:
This is copied from the original implementation. I changed it to a relatively better one.